modules/conductor/src/main/java/org/opencastproject/event/handler/OaiPmhUpdatedEventHandler.java

```diff
@@ -366,111 +135,85 @@ public void handleEvent(AssetManagerItem.TakeSnapshot snapshotItem) {
       return;
     }
 
+    if (AssetManagerItem.Type.Update != snapshotItem.getType()) {
+      logger.trace("Handling the event type {} is not implemented yet", snapshotItem.getType().toString());
+      return;
+    }
+
```

Comment:
The subclass TakeSnapshot of AssetManagerItem is always of Type Update, so it's unneccessary to check the type here. :)

Reply:
Oh yes, thank you. This check is unneeded. I will remove it.